conftest.py

```diff
@@ -119,16 +124,68 @@ def pytest_runtest_logreport(self, report):
         line = self._locationline(rep.nodeid, *rep.location)
         if not hasattr(rep, 'node'):
             word = "({})".format(word)
-            if len(rep.nodeid.split("::")) >= 3:
-                indentation = "    "
-            else:
-                indentation = "  "
+            indentation = self._add_indentation_for_tests_list_item(rep)
             self.write_ensure_prefix(indentation + test_title, word, **markup)
-            #self._tw.write(word, **markup)
         else:
             self.ensure_newline()
             if hasattr(rep, 'node'):
                 self._tw.write("[%s] " % rep.node.gateway.id)
```

Comment:
You've got a mix of  `%` string formatting and `.format` string formatting. It would be good to make this consistent. I think `.format` is the favoured way but you could look at how it's done elsewhere in pytest, I suppose.